.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
```

Comment:
Disabling the blank issue might be controversial, because it forces all users to use the templates and loose the flexibility. I understand that you may be intentionally doing this, but let's revisit this after we get more feedback from the users/community.